core/cloudflow-operator/src/main/scala/cloudflow/operator/CloudflowLabels.scala

```diff
@@ -49,6 +49,12 @@ object CloudflowLabels {
   // The git tag of the user application
   val Version = "app.kubernetes.io/version"
 
+  val ConfigFormat = "com.lightbend.cloudflow/config-format"
```

Comment:
A label that indicates the format of the secret, input (merged and resembling format of what the user uses) or output (the processed format that works as an application.conf)